models/quantal_stp_connection.h

```diff
@@ -212,7 +212,7 @@ Quantal_StpConnection< targetidentifierT >::send( Event& e,
   // Compute number of sites that recovered during the interval.
   for ( int depleted = n_ - a_; depleted > 0; --depleted )
   {
-    if ( kernel().rng_manager.get_rng( vp )->drand() < ( 1.0 - p_decay ) )
+    if ( kernel().rng_manager.get_rng( t )->drand() < ( 1.0 - p_decay ) )
     {
```

Comment:
You can now remove line 200 of this file where the virtual process id is retrieved since it is now longer needed.